content/issues/2021-06-07-repo-jenkins-ci-org.md

```diff
@@ -1,14 +1,23 @@
 ---
 title: repo.jenkins-ci.org issues
 date: 2021-06-07 12:37:00
-resolvedWhen: 
-resolved: false
+resolvedWhen: 2021-06-07 16:37:00
+resolved: true
 # Possible severity levels: down, disrupted, notice
 severity: disrupted
 affected:
   - repo.jenkins-ci.org
 section: issue
 ---
+**2021-06-08 07:37:00**
+
+This [PR](https://github.com/jenkins-infra/charts/pull/1238) persist in time the hotfix we applied.
+
+**2021-06-07 16:37:00**
+
+Two need IPs have been manually added to ldap.jenkins.io configuration.
```

Comment:
```suggestion
Two IPs have been manually added to ldap.jenkins.io configuration.
```

Reply:
Thanks, definitely lack of sleep :(,  I meant Two new IPs